### PR TITLE
Send to BTC Address: Correctly calculate fees based on chosen feerate

### LINKED ISSUE #791: Send to BTC Address: Correctly calculate fees based on chosen feerate
Currently, the "Send to BTC Address" is incorrectly using `prepare_redeem_onchain_funds` to calculate the reverse swap fees.

It should instead use `fetch_reverse_swap_fees` and https://github.com/breez/breez-sdk/issues/801 to calculate the actual reverse swap fees.

### PR DESCRIPTION
**Pull request by erdemyerebasmaz**
Closes #791

### CODE REVIEW

**Comment by roeierez** [comment]
Looks good, we just need to wait for @ok300 to add the optional address parameter to the reverse swap fees API.

**Comment by ok300** [comment]
@roeierez  As I started the implementation, I found that I didn't need the optional address field for the claim tx feerate, as we're using a fixed approximation of the script size.

My PR is ready for review: https://github.com/breez/breez-sdk/pull/802

File: lib/routes/withdraw/widgets/fee_chooser/widgets/fee_breakdown.dart [hunk 1]
```diff
@@ -47,15 +55,14 @@ class FeeBreakdown extends StatelessWidget {
...
-                  sendAmount = swapAmount;
-                } else {
-                  sendAmount = swapAmount;
-                  if (boltzFees != null) {
-                    sendAmount += boltzFees!;
+                int sendAmount = swapAmount;
+                if (isMaxValue != true) {
+                  if (pairInfo != null) {
+                    var p = pairInfo!.feesPercentage / 100;
+                    var boltzFees = ((swapAmount + pairInfo!.feesClaim).toDouble() * p / (1 - p) +
```
Comment (ok300) [inline lib/routes/withdraw/widgets/fee_chooser/widgets/fee_breakdown.dart:62]: Same here. Maybe extract this in a method, if it's re-used?

File: lib/routes/withdraw/widgets/fee_chooser/widgets/fee_breakdown.dart [hunk 2]
```diff
@@ -26,6 +27,13 @@ class FeeBreakdown extends StatelessWidget {
     final texts = context.texts();
     final themeData = Theme.of(context);
     final minFont = MinFontSize(context);
+    int? boltzFees;
+    if (pairInfo != null) {
+      var p = pairInfo!.feesPercentage / 100;
+      boltzFees = ((swapAmount + pairInfo!.feesClaim).toDouble() * p / (1 - p) +
```
Comment (ok300) [inline lib/routes/withdraw/widgets/fee_chooser/widgets/fee_breakdown.dart:33]: Its a bit hard to follow. Can you break this down in intermediary steps?

**Comment by erdemyerebasmaz** [comment]
Closed in favor of #796
